lib/grammers-client/src/client/chats.rs

```diff
@@ -725,6 +725,55 @@ impl Client {
         let permissions = ParticipantPermissions::Channel(participant.participant);
         Ok(permissions)
     }
+    
+    /// join private chat
+    pub async fn accept_invite_link(
+        &mut self,
+        invite_link: &str,
+    ) -> Result<Option<Chat>, InvocationError> {
+        use tl::enums::Updates;
+        assert!(invite_link.starts_with("https://t.me/joinchat/"));
+        let update_chat = match self
+            .invoke(&tl::functions::messages::ImportChatInvite {
+                hash: invite_link.replace("https://t.me/joinchat/", ""),
+            })
+            .await?
+        {
+            Updates::Combined(updates) => updates.chats.first().cloned(),
+            Updates::Updates(updates) => updates.chats.first().cloned(),
+            _ => None,
+        };
+
+        if let Some(chat) = update_chat {
+            return Ok(Some(Chat::from_chat(chat)));
+        }
+        Ok(None)
```

Comment:
Can this actually happen?

Reply:
It's ok, is there any problem here?

Reply:
If this can never occur, I would prefer `update_chat` being `None` to `panic!()`. This way the method can always return the chat and not just an `Option`.

If this can occur, we should document when it can happen.